src/main/java/com/psk/pyragas/ToolRent/usecases/Login.java

```diff
@@ -1,36 +1,43 @@
 package com.psk.pyragas.ToolRent.usecases;
 import com.psk.pyragas.ToolRent.dao.ProfilesDAO;
+import com.psk.pyragas.ToolRent.entities.Profile;
 import lombok.Getter;
 import lombok.Setter;
 
+import javax.enterprise.context.ApplicationScoped;
 import javax.enterprise.context.SessionScoped;
+import javax.inject.Inject;
 import javax.inject.Named;
 import java.io.Serializable;
+import java.util.List;
 
 @Getter
 @Setter
 @Named
-@SessionScoped
+@ApplicationScoped
 public class Login implements Serializable {
 
-    @Getter
-    private boolean isLoggedIn;
+    @Inject
+    private ProfilesDAO profilesDAO;
+
+    @Getter @Setter
```

Comment:
These are not needed. Or maybe these are all you need, so you can remove the ones on top of the class.

Reply:
Probably only leave the annotations on top of the class.